gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -601,4 +605,17 @@ def run(argv=None):
 
 if __name__ == '__main__':
   logging.getLogger().setLevel(logging.INFO)
-  run()
+  try:
+    run()
+  except Exception as e:
+    if _newly_created_tables:
```

Comment:
I think it's possible to not have any tables in _newly_created_tables but still have append off. This would happen if errors happens before any table is generated.

Reply:
I agree, but it was getting too complicated to pass `known_args.append` here.
So I decided to show this warning just based on empty `_newly_created_tables` (which means in some cases it's wrong).